proto/options.proto

```diff
@@ -307,6 +307,9 @@ message Costing {
     oneof has_elevator_penalty {
       float elevator_penalty = 79;
     }
+    oneof has_fixed_speed {
+      float fixed_speed = 80;
+    }
```

Comment:
since you use a default of `0` you can simply make this:

```suggestion
    float fixed_speed = 80;
```

Reply:
When I changed to that format I got a compile error saying the member didn't exist in costing options so I left it as it was.

Reply:
im sure we can fix that, we want to have as few `oneof`'s as possible. can you post back the compiler error?